test-integration/scenarios/loans/1_eth_takeout_loan_and_2_repays_full_successfully.js

```diff
@@ -0,0 +1,153 @@
+// Util classes
+const {teller, tokens} = require("../../../scripts/utils/contracts");
+const {
+  loans: loansActions,
+  oracles: oraclesActions,
```

Comment:
I think this is an old version. Now the filename doesn't contain the collateral token name. Please verify it with the current IT on the develop branch.